gamemodes/terrortown/gamemode/shared/lang/pl.lua

```diff
@@ -6,7 +6,7 @@ local L = LANG.CreateLanguage("pl")
 -- the alias name is based on the original TTT language name:
 -- - does not exist -
 L.__alias = "polski"
-
+-- If you having problems with polish ó ł ż ź ą ę please use alias 	"extended = true," in vgui files that contains fonts, especially in surface.CreateFont
```

Comment:
I'm not exactly sure what this means, but for continuity reasons I'd like you to ask to not add a line here. All language files should have the same lines.